modules/aws/win-gfx/win-gfx-startup.ps1.tmpl

```diff
@@ -60,6 +65,50 @@ function Retry([scriptblock]$Action, $Interval = 10, $Attempts = 30) {
   }
 }
 
+function Nvidia-is-Installed {
+    if (!(test-path $NVIDIA_DIR)) {
+        return $false
+    }
+
+    cd $NVIDIA_DIR
+    & .\nvidia-smi.exe
+    return $?
+    return $false

```

Comment:
why are there 2 returns here?

Reply:
I reused the same function as GCP, should I make a separate commit to remove both?

Reply:
can you confirm what the return $_ gives you in both cases (before and after install)

The 2nd line should not be there. The GCP version needs to fix this too as it will never reach the 2nd line.

Reply:
I'll correct this and force update this branch.

Tested by calling the function before and after:
```
Nvidia-is-Installed
Nvidia-Install
Nvidia-is-Installed
```

Result:
```
False
################################################################
Install NVIDIA GRID Driver
################################################################
<driver install log omitted here>

Nvidia-is-installed True
True